tests/wpt/mozilla/tests/mozilla/referrer-policy/generic/common.js

```diff
@@ -194,9 +194,10 @@ function queryAnchor(url, callback, referrer_policy) {
     var id = token();
     var url_with_params = url + "&id=" + id + "&tagAttrs=" + JSON.stringify(referrer_policy);
     var iframe = appendIframeToBody(url_with_params);
+    var first_load = true;
     iframe.addEventListener("load", function listener() {
-      if ((iframe.contentWindow !== null) &&
-          (iframe.contentWindow.location.toString() === url_with_params)) {
+      if (first_load) {
+        first_load = false;
```

Comment:
I believe the test loads a page in the iframe which then navigates again and we wait for the second load event before doing the xmlhttp request. I could be wrong, as I am not entirely certain how the referrer tests work. This change gave the same behavior on all of the already passing tests; however caused the ones relying on https to go from `ERROR` to `TIMEOUT`.